### PR TITLE
Add more changes for kifi site release

### PR DESCRIPTION
**Pull request by gmethvin**

### CODE REVIEW

File: packrat/scripts/slider2.js [hunk 1]
```diff
@@ -578,7 +578,7 @@ slider2 = function() {
               var q, el = this;
               if (e.which == 13 && (q = el.value.trim())) {
                 api.port.emit("session", function(session) {
-                  var uri = session && ~session.experiments.indexOf("website") ? "https://www.kifi.com/site/search?q=" : "https://www.google.com/search?q=";
+                  var uri = session ? "https://www.kifi.com/site/search?q=" : "https://www.google.com/search?q=";
```
Comment (2is10) [inline packrat/scripts/slider2.js:581]: We should remove `/site` from the URL and, in my opinion, also remove the session check now. As of last week, the pane is removed when the user logs out, and a session is required to open the pane.

File: kifi-backend/modules/shoebox/public/assets [hunk 2]
```diff
@@ -0,0 +1 @@
+.
```
Comment (2is10) [inline kifi-backend/modules/shoebox/public/assets:1]: What’s this file for?

File: kifi-backend/modules/shoebox/assets [hunk 3]
```diff
@@ -0,0 +1 @@
+assets
```
Comment (2is10) [inline kifi-backend/modules/shoebox/assets:1]: And this file? Is it for local non-play dev? What does it accomplish?

**Comment by 2is10** [comment]
Looks good. Thanks for adding the redirect and removing the experiment.

File: packrat/scripts/slider2.js (hunk 1 above)
Comment (gmethvin) [inline packrat/scripts/slider2.js:581]: obviously, I'll fix it

File: kifi-backend/modules/shoebox/public/assets (hunk 2 above)
Comment (gmethvin) [inline kifi-backend/modules/shoebox/public/assets:1]: this is for local dev

File: kifi-backend/modules/shoebox/assets (hunk 3 above)
Comment (gmethvin) [inline kifi-backend/modules/shoebox/assets:1]: this one is an extra one I accidentally added; I removed it later

**Pull request merged**